src/data/languages/types.ts

```diff
@@ -0,0 +1,30 @@
+const languageKeys = [
```

Comment:
[question] Could this be derived from `languageInfo.ts`? Then we have a single source of truth :)

Reply:
As it stands this would be a circular dependency as the object in `languageInfo` is typed from this - but tbh that's overtyped, we can just do as you suggest